.travis.yml

```diff
@@ -56,6 +56,14 @@ jobs:
         - source script/travis-awscli-installation
         - integration/js/script/install-kite
       script: npm run test:integration-content-share
+    #Integ test content share
```

Comment:
Typo in comment, but also I don't think we need these comments as they are redundant

Reply:
Fixed